frontend/frontend-react/src/components/template/Main.css

```diff
@@ -1,31 +1,33 @@
 div.content {
-    background-color: #fafafa;
-    
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    
     grid-area: content;
+    background-color: #dadbda;
 }
 
 .icons {
     margin-right: 30px;
 }
 
 .post {
-    height: 550px;
-    width: 550px;
-
+    max-width: 500px;
     display: flex;
-    justify-content: space-around;
-    align-items: center;
     flex-direction: column;
 
+    border-radius: 4px;
     background-color: #fffefe;
+    border: 1.5px solid #dadbda;
 }
 
 .icon {
     width: 50px;
     height: 50px;
 
+    background-image: url("../../assets/WhatsApp\ Image\ 2021-06-03\ at\ 12.21.36.jpeg");
```

Comment:
This image will only work on your machine. As I expected it didn't work on my machine.

You should include the file into the project or upload it somewhere like https://imgur.com/

![Screenshot from 2021-06-15 21-11-13](https://user-images.githubusercontent.com/54550926/122139065-77d94300-ce1e-11eb-8b73-45d297361028.png)

Reply:
We forgot to commit the image kkkk, but we just commited it.

Reply:
Hahahaha it happens 🚀